pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -316,6 +319,85 @@ func StartVirtlog(stopChan chan struct{}, domainName string) {
 			time.Sleep(time.Second)
 		}
 	}()
+
+	go func() {
+		const readSerialFilename = "readserial.py"
+		const readSerialScript = "/etc/qemu/scripts/" + readSerialFilename
```

Comment:
Do we really need two separate consts?  The other is used only for printing and even there maybe it's better to print the full path ?

Reply:
Sure, I'll print the full path. thanks